### ISSUE TITLE
Critical Issue: an active PHP session was detected | (caused by Unyson)

### ISSUE THREAD

**Issue by nextfilmz** [ThemeFuse/Unyson#4032]
This critical issue was displayed in the Site Health Status on wordpress:

"A PHP session was created by a session_start() function call. This interferes with REST API and loopback requests. The session should be closed by session_write_close() before making any HTTP requests."

When I deactivate Unyson, the issue goes away.

How do I fix this? I would appreciate any help.

Thank you

**Comment by NasimNet** [comment]
You can temporarily remove the error by adding the following code to your theme

```
function disable_fw_use_sessions() {
	return false;
}
add_filter( 'fw_use_sessions', 'disable_fw_use_sessions' );
```

**Comment by CarlosSLoureiro** [comment]
@NasimNet This i'll cause the flash messages to not be printed.
_at "/framework/helpers/class-fw-flash-messages.php" - line 116_

Apparently it was added to fix another issue: [#2219](https://github.com/ThemeFuse/Unyson/issues/2219)

**Comment by NasimNet** [comment]
@CarlosSLoureiro 
I'm not sure, but I do not think so!
This is a temporary cache only
The issue you submitted is very old and has nothing to do with it

**Comment by Kursymru** [comment]
Dear all,

I am facing the same issue too, I brought a theme which required Unyson. When Unyson plugin is activated the website blocked on the preloader. Any idea how to fix this, please?

Thanks
Kursy

**Comment by ojosazules4228** [comment]
Same problem here! Any tip on how to fix it?

Thanks!

**Comment by otakuyose** [comment]
The problem is not only because of the message, it turns out that when I deactivate the unyson plugin my page loads much faster, before this problem appeared, my page was loading fast, then everything is too slow

**Comment by corzel** [comment]
I fixed it adding: 
**session_write_close();** 
in lines 231 and 262 in this file: wp-content/plugins/unyson/framework/includes/hooks.php
```
229: session_start();
230: }
231: session_write_close();
232: }

**Comment by nildopio** [comment]
> I fixed it adding:
> **session_write_close();**
> in lines 231 and 262 in this file: wp-content/plugins/unyson/framework/includes/hooks.php
> 
> ```
> 229: session_start();
> 230: }
> 231: session_write_close();
> 232: }
> ```

Hi... can you explain this solution?

**Comment by mcworks** [comment]
> 
> 
> I fixed it adding:
> **session_write_close();**
> in lines 231 and 262 in this file: wp-content/plugins/unyson/framework/includes/hooks.php
> 
> ```
> 229: session_start();
> 230: }
> 231: session_write_close();
> 232: }
> ```

Thanks this sorted me out...
